packages/suite/src/views/wallet/transactions/components/TransactionSummary/components/SummaryCards/components/InfoCard/index.tsx

```diff
@@ -47,7 +47,6 @@ const SecondaryValueWrapper = styled.div`
     font-variant-numeric: tabular-nums;
     /* margin-left: 1ch; */
     line-height: 1.57;
-    white-space: nowrap;
```

Comment:
I would avoid separating date on more lines.

Instead of 
Feb 10, 2021 - Apr 17, 
2021

I would do
Feb 10, 2021 -
Apr 17, 2021

Reply:
great idea, but I'm afraid that it's surprisingly hard to achive with react-intl :-(

Reply:
It is created from two `FormattedDate` components, so I think it should not be a problem

Reply:
Fixed in https://github.com/trezor/trezor-suite/pull/3887/commits/6cfc57127c9e4c265ae91ea484333ac72c439c76